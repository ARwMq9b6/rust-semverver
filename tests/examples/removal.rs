mod old {
    pub struct Abc {}
}

mod new {

}
