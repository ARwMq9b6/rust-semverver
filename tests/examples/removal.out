warning: struct is never used: `Abc`
 --> tests/examples/basic.rs:2:5
  |
2 |     pub struct Abc {}
  |     ^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(dead_code)] on by default

thread 'rustc' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:335
stack backtrace:
   0:     0x7f139e9b03e3 - std::sys::imp::backtrace::tracing::imp::unwind_backtrace::hcab99e0793da62c7
                               at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1:     0x7f139e9ab39a - std::sys_common::backtrace::_print::hbfe5b0c7e79c0711
                               at /checkout/src/libstd/sys_common/backtrace.rs:71
   2:     0x7f139e9bf1aa - std::panicking::default_hook::{{closure}}::h9ba2c6973907a2be
                               at /checkout/src/libstd/sys_common/backtrace.rs:60
                               at /checkout/src/libstd/panicking.rs:355
   3:     0x7f139e9bedab - std::panicking::default_hook::he4d55e2dd21c3cca
                               at /checkout/src/libstd/panicking.rs:371
   4:     0x7f139e9bf5bb - std::panicking::rust_panic_with_hook::ha138c05cd33ad44d
                               at /checkout/src/libstd/panicking.rs:549
   5:     0x7f139e9bf494 - std::panicking::begin_panic::hcdbfa35c94142fa2
                               at /checkout/src/libstd/panicking.rs:511
   6:     0x7f139e9bf3c9 - std::panicking::begin_panic_fmt::hc09fe500d9b7be81
                               at /checkout/src/libstd/panicking.rs:495
   7:     0x7f139e9bf357 - rust_begin_unwind
                               at /checkout/src/libstd/panicking.rs:471
   8:     0x7f139ea0762d - core::panicking::panic_fmt::h883a028e9f4b4457
                               at /checkout/src/libcore/panicking.rs:69
   9:     0x7f139ea07564 - core::panicking::panic::hdb3cf3207dda37bb
                               at /checkout/src/libcore/panicking.rs:49
  10:     0x7f139d54c5bd - rustc_metadata::cstore::CStore::get_crate_data::h445ab0320781c30b
  11:     0x7f139d539c44 - rustc_metadata::cstore_impl::<impl rustc::middle::cstore::CrateStore for rustc_metadata::cstore::CStore>::item_children::h81f6f098556fec3e
  12:     0x55d71974b93a - rust_semverver::callback::h961cda43d644d484
                               at /home/twk/gsoc/rust-semverver/src/bin/rust_semverver.rs:68
  13:     0x55d71974c430 - <rust_semverver::SemVerVerCompilerCalls as rustc_driver::CompilerCalls<'a>>::build_controller::{{closure}}::hbf52b5b070796df8
                               at /home/twk/gsoc/rust-semverver/src/bin/rust_semverver.rs:151
  14:     0x7f139f7d0900 - rustc_driver::driver::compile_input::{{closure}}::h5dd0b6da256c9f40
  15:     0x7f139f7f4c8c - rustc_driver::driver::phase_3_run_analysis_passes::{{closure}}::h5ee2deada840296d
  16:     0x7f139f7ef21b - rustc_driver::driver::phase_3_run_analysis_passes::hbc88dd193ceb02a2
  17:     0x7f139f7cd860 - rustc_driver::driver::compile_input::h22316d34abd2e92e
  18:     0x7f139f816086 - rustc_driver::run_compiler::he2000b3a5ab5e49a
  19:     0x55d71974ccc3 - rust_semverver::main::{{closure}}::hfa63611fbdf249ba
                               at /home/twk/gsoc/rust-semverver/src/bin/rust_semverver.rs:193
  20:     0x55d71971ca6a - std::sys_common::backtrace::__rust_begin_short_backtrace::h76a8599ef2491d7a
                               at /checkout/src/libstd/sys_common/backtrace.rs:136
  21:     0x55d71971d90a - std::thread::Builder::spawn::{{closure}}::{{closure}}::h90f183c75e2a9e24
                               at /checkout/src/libstd/thread/mod.rs:364
  22:     0x55d7197487fa - <std::panic::AssertUnwindSafe<F> as core::ops::FnOnce<()>>::call_once::h8efd35a7243e084f
                               at /checkout/src/libstd/panic.rs:296
  23:     0x55d71971de67 - std::panicking::try::do_call::h7e716e63e0d08d1a
                               at /checkout/src/libstd/panicking.rs:454
  24:     0x7f139e9c890a - __rust_maybe_catch_panic
                               at /checkout/src/libpanic_unwind/lib.rs:98
  25:     0x55d71971dd1d - std::panicking::try::h310202eb16363da5
                               at /checkout/src/libstd/panicking.rs:433
  26:     0x55d71971cf01 - std::panic::catch_unwind::h346b7cad9419fbb9
                               at /checkout/src/libstd/panic.rs:361
  27:     0x55d71971d76a - std::thread::Builder::spawn::{{closure}}::h2d8ab74385184d58
                               at /checkout/src/libstd/thread/mod.rs:363
  28:     0x55d7197361c7 - <F as alloc::boxed::FnBox<A>>::call_box::hb62016c613268c9c
                               at /checkout/src/liballoc/boxed.rs:648
  29:     0x7f139e9bdfb5 - std::sys::imp::thread::Thread::new::thread_start::h227b2afaa9316a8d
                               at /checkout/src/liballoc/boxed.rs:658
                               at /checkout/src/libstd/sys_common/thread.rs:21
                               at /checkout/src/libstd/sys/unix/thread.rs:84
  30:     0x7f139e717213 - start_thread
                               at /builddir/glibc-2.25/nptl/pthread_create.c:456
  31:     0x7f139e24316e - clone
                               at ../sysdeps/unix/sysv/linux/x86_64/clone.S:97
  32:                0x0 - <unknown>
thread 'main' panicked at 'rustc thread failed: Any', /checkout/src/libcore/result.rs:859
stack backtrace:
   0:     0x7f139e9b03e3 - std::sys::imp::backtrace::tracing::imp::unwind_backtrace::hcab99e0793da62c7
                               at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1:     0x7f139e9ab39a - std::sys_common::backtrace::_print::hbfe5b0c7e79c0711
                               at /checkout/src/libstd/sys_common/backtrace.rs:71
   2:     0x7f139e9bf1aa - std::panicking::default_hook::{{closure}}::h9ba2c6973907a2be
                               at /checkout/src/libstd/sys_common/backtrace.rs:60
                               at /checkout/src/libstd/panicking.rs:355
   3:     0x7f139e9bedab - std::panicking::default_hook::he4d55e2dd21c3cca
                               at /checkout/src/libstd/panicking.rs:371
   4:     0x7f139e9bf5bb - std::panicking::rust_panic_with_hook::ha138c05cd33ad44d
                               at /checkout/src/libstd/panicking.rs:549
   5:     0x7f139e9bf494 - std::panicking::begin_panic::hcdbfa35c94142fa2
                               at /checkout/src/libstd/panicking.rs:511
   6:     0x7f139e9bf3c9 - std::panicking::begin_panic_fmt::hc09fe500d9b7be81
                               at /checkout/src/libstd/panicking.rs:495
   7:     0x7f139e9bf357 - rust_begin_unwind
                               at /checkout/src/libstd/panicking.rs:471
   8:     0x7f139ea0762d - core::panicking::panic_fmt::h883a028e9f4b4457
                               at /checkout/src/libcore/panicking.rs:69
   9:     0x55d71973617a - core::result::unwrap_failed::h834326e7b6af2497
                               at /checkout/src/libcore/macros.rs:29
  10:     0x55d7197212d6 - <core::result::Result<T, E>>::expect::h09bfbddf509147ed
                               at /checkout/src/libcore/result.rs:761
  11:     0x55d71974c70c - rust_semverver::main::hec9b0208f30ee0af
                               at /home/twk/gsoc/rust-semverver/src/bin/rust_semverver.rs:182
  12:     0x7f139e9c890a - __rust_maybe_catch_panic
                               at /checkout/src/libpanic_unwind/lib.rs:98
  13:     0x7f139e9c0458 - std::rt::lang_start::ha09816a4e25587ea
                               at /checkout/src/libstd/panicking.rs:433
                               at /checkout/src/libstd/panic.rs:361
                               at /checkout/src/libstd/rt.rs:59
  14:     0x55d71974cdf2 - main
  15:     0x7f139e17b4e0 - __libc_start_main
                               at ../csu/libc-start.c:295
  16:     0x55d71970be09 - _start
                               at ../sysdeps/x86_64/start.S:120
  17:                0x0 - <unknown>
